pkg/daemon/update.go

```diff
@@ -865,22 +865,43 @@ func generateExtensionsArgs(oldConfig, newConfig *mcfgv1.MachineConfig) []string
 		}
 	}
 
+	// Supported extensions has package list info that is required
+	// to enable an extension
+	extensions := getSupportedExtensions()
```

Comment:
This breaks OKD 4.7 :(

Reply:
Can you link me to that code?   Wait...I think I found it:
https://github.com/openshift/release/blob/959c2954344438c4eed3ec7f52a5e099e8335516/ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml#L586

That's...terrifying :smile: 

I guess it's hard to do much here other than continue to entirely remove validation on OKD short term.

Reply:
Right, previously MCC verified that extensions list is valid (on FCOS it skipped this check). Guess its time to rework it so that MCD could mutate it (and leave as is on OKD)

Reply:
damm, missed this usecase. It we will good to have okd-gcp-op test running so that we don't break it.
Will give some thought to fix it.